dynosaur/examples/trait_variant.rs

```diff
@@ -0,0 +1,107 @@
+#[trait_variant::make(SendNext: Send)]
+#[dynosaur::dynosaur(DynNext = dyn Next, blanket = none)]
+#[dynosaur::dynosaur(DynSendNext = dyn SendNext, blanket = none)]
```

Comment:
I believe this should work with `blanket = dyn` on both.

Reply:
Fixed